libs/geo/src/main/java/org/elasticsearch/geo/utils/WellKnownText.java

```diff
@@ -52,12 +53,20 @@
     public static final String COMMA = ",";
     public static final String NAN = "NaN";
 
-    private static final String NUMBER = "<NUMBER>";
-    private static final String EOF = "END-OF-STREAM";
-    private static final String EOL = "END-OF-LINE";
+    private final String NUMBER = "<NUMBER>";
+    private final String EOF = "END-OF-STREAM";
+    private final String EOL = "END-OF-LINE";
 
-    public WellKnownText() {
+    private final boolean coerce;
+    private final boolean ignoreZValue;
 
+    public WellKnownText(boolean coerce, boolean ignoreZValue) {
+        this.coerce = coerce;
+        this.ignoreZValue = ignoreZValue;
+    }
+
+    public WellKnownText() {
```

Comment:
is this needed?

Reply:
Yeah, that's probably harmful now. I will remove it.